types/time.go

```diff
@@ -18,63 +17,43 @@ type Time time.Time
 func (t *Time) UnmarshalJSON(data []byte) error {
 	s := string(data)
 
-	switch s {
-	case "null", "0", `""`, `"0"`:
-		*t = Time(time.Time{})
-		return nil
-	}
-
 	if s[0] == '"' {
 		s = s[1 : len(s)-1]
 	}
 
-	badSyntax := false
-	target := strings.IndexFunc(s, func(r rune) bool {
-		if r == '.' {
-			return true
-		}
-		// types.Time may only parse numbers. The below check ensures an error is thrown. time.Time should be used to
-		// parse RFC3339 strings instead.
-		badSyntax = r < '0' || r > '9'
-		return badSyntax
-	})
+	switch s {
+	case "null", "0", ``:
```

Comment:
🦍 Think backticks were left over from before and we can just consistently use quotation marks.
```suggestion
	case "null", "0", "":
```

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/1912/commits/4973ffb9ec6d9488f18e9f67ef32577157da8283 👀 😆